docker-compose.yml

```diff
@@ -20,11 +27,17 @@ services:
       replicas: 1
     depends_on:
       - eventstore
+      - mongo
     ports:
       - 3000:3000
     environment:
       - PORT=3000
-      - NODE_ID="1"
+      - NODE_ID=node_1
+      - MONGO_HOST=mongo
+      - MONGO_PORT=27017
+      - MONGO_DATABASE=owners
```

Comment:
`owners` is niet helemaal dekkend, de sensoren e.d. zitten hier immers ook in. Kan hernoemd worden naar `sensrnet` of `sensrnet-registry` oid

Reply:
goed idee